docs/getting-started/walk-through/advanced-hera-features.md

```diff
@@ -113,22 +113,23 @@ global_config.experimental_features["script_runner"] = True
 ```
 
 
-#### Script Annotations
-Annotations are supported for input `Parameter`s and `Artifact`s. They can be added using `typing.Annotated` in the function parameters. 
+### Script Annotations
 
-```python
-@script()
-def func(a_param: Annotated[str, Parameter(name="a-param", default="param")]):
-    ...
-```
+Annotation syntax using `typing.Annotated` is supported for `Parameter`s and `Artifact`s as inputs and outputs for
+functions decorated as `scripts`. They use `Annotated` as the type in the function parameters and allow us to simplify
+writing scripts with parameters and artifacts that require additional fields such as a `description` or alternative
+`name`.
 
-This allows us to simplify writing scripts with parameters and artifacts that require additional fields. 
 
-An example can be seen [here](../../examples/workflows/script_annoations_inputs.md)
+This feature can be enabled by setting the `experimental_feature` flag `script_annotations`
 
-##### Parameters
+```py
+global_config.experimental_features["script_annotations"] = True
+```
 
-In Hera, we can specify inputs inside the `@script` decorator as follows:
+#### Parameters
+
+In Hera, we can currently specify inputs inside the `@script` decorator as follows:
```

Comment:
I think "currently" is not the best word here, since both the "old" and the "new" are currently supported

Reply:
"currently supported" as in, "doesn't require you to switch on the experimental flag" - not sure there's a better, more succinct word

Reply:
Ah, I see. Maybe just drop it altogether? 
"In Hera, we can specify inputs inside the `@script` decorator as follows:"

Reply:
That is a good point, it makes it easier to contrast them